CONTRIBUTING.md

```diff
@@ -54,6 +54,20 @@ The sources for the [Coq reference manual](https://coq.inria.fr/distrib/current/
 
 You may also contribute to the informal documentation available in [Cocorico](https://github.com/coq/coq/wiki) (the Coq wiki), and the [Coq FAQ](https://github.com/coq/coq/wiki/The-Coq-FAQ). Both of these are editable by anyone with a GitHub account.
 
+## Following the development
+
+If you want to follow the development activity around Coq, you are encouraged
+to subscribe to the [Coqdev mailing list](https://sympa.inria.fr/sympa/info/coqdev).
+The activity on this mailing list is quite reasonable.
```

Comment:
I'm not sure how explicit "quite reasonable" is. Maybe "has reasonably low traffic" or something like that instead?